src/space_flight/pipelines/data_engineering/nodes.py

```diff
@@ -101,3 +101,37 @@ def create_master_table(
     master_table = with_companies.drop(["shuttle_id", "company_id"], axis=1)
     master_table = master_table.dropna()
     return master_table
+
+
+# Implementation of a sorting algorithm
+# Source: https://bosker.wordpress.com/2011/04/29/the-worst-algorithm-in-the-world/
+def _slowsort(values, i, j):
+    if i >= j:
+        return
+
+    # get the middle index
+    m = int((i + j) / 2)
+
+    # sort both sides of the middle of this index
+    _slowsort(values, i, m)
+    _slowsort(values, m + 1, j)
+
+    # if the RHS is less than the middle, swap places
+    if values[j] < values[m]:
+        k = values[m]
+        values[m] = values[j]
+        values[j] = k
+
+    # continue sorting (slowly!)
+    _slowsort(values, i, j - 1)
```

Comment:
I find it hard to remember the meanings of the variables using initials. Could you write them out as meaningful variable names to make it easier to read? #nitpick1000

Reply:
Good catch. I'll make the changes now to make the variable names into actual words and not just letters